db/postgres/migrations/1715595241211-AlterResetPasswordRequestExpiresAtNullableFalse.ts

```diff
@@ -0,0 +1,14 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AlterResetPasswordRequestExpiresAtNullableFalse1715595241211 implements MigrationInterface {
+    name = 'AlterResetPasswordRequestExpiresAtNullableFalse1715595241211'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "users_reset_password_requests" ALTER COLUMN "expires_at" SET NOT NULL`);
```

Comment:
This migration may not work in a project where the service is actively used. Not NULL works if there is no data in the table, but if there is data, there may be an error in migration transition. Can you test this in detail and check the output?

Reply:
imo that is not migration concern. because it may involve business decisions and deep nested data decisions and it is not migrations responsibility. it should be manual process on merge to environments